iOSOverUsbSupport.cs

```diff
@@ -159,6 +160,9 @@ class PosixDllLoader: IDllLoader
 
 		public IntPtr LoadLibrary(string fileName)
 		{
+			if (!File.Exists (fileName))
+				throw new FileNotFoundException (fileName);
```

Comment:
Yes, we are passing a path here, code path starts here:

https://github.com/Unity-Technologies/MonoDevelop.Debugger.Soft.Unity/blob/unity-staging/iOSOverUsbSupport.cs#L328

I made this change to able to distinguish between dlopen related errors and the editor path not being set up.